### ISSUE TITLE
Failing to compile any kernel on Fedora Rawhide 

### ISSUE THREAD

**Issue by BrycensRanch** [Frogging-Family/linux-tkg#967]
[shell-output.log.txt](https://github.com/user-attachments/files/16146770/shell-output.log.txt)
[customization.cfg.txt](https://github.com/user-attachments/files/16146771/customization.cfg.txt)
[prepare.log.txt](https://github.com/user-attachments/files/16146772/prepare.log.txt)
[system-info.txt](https://github.com/user-attachments/files/16146773/system-info.txt)

```bash
❯ fastfetch --logo none
romvnly@Shadow-PH315-54
-----------------------
OS: Fedora Linux 41 (KDE Plasma) x86_64
Host: Predator PH315-54 (V1.15)
Kernel: Linux 6.10.0-0.rc6.50.fc41.x86_64
Uptime: 17 hours, 53 mins
Packages: 4507 (rpm), 48 (nix-default), 40 (flatpak-system), 2 (flatpak-user), 7 (snap), 12 (brew)
Shell: bash 5.2.26
Display (BOE08B3): 1920x1080 @ 144Hz [Built-in]
DE: KDE Plasma 6.1.2
WM: KWin (Wayland)
WM Theme: WillowDarkBlur
Theme: Breeze (AbsoluteDark) [QT], Colorful-Dark-GTK [GTK3/4]
Icons: Windows11-black-dark [QT], Windows11-black-dark [GTK3/4]
Font: SegoeUI Nerd Font (11pt) [QT], SegoeUI Nerd Font (11pt) [GTK3/4]
Cursor: WindowsVista (48px)
Terminal: kitty 0.35.2
Terminal Font: CascadiaMonoNF-Regular (11pt)
CPU: 11th Gen Intel(R) Core(TM) i7-11800H (16) @ 4.60 GHz
GPU 1: Intel UHD Graphics @ 1.45 GHz [Integrated]
GPU 2: NVIDIA GeForce RTX 3060 Mobile / Max-Q [Discrete]
Memory: 8.52 GiB / 15.39 GiB (55%)
Swap: 6.62 GiB / 22.35 GiB (30%)
Disk (/): 430.27 GiB / 474.64 GiB (91%) - btrfs
Battery: 100% [AC Connected]
Locale: en_US.UTF-8
```

**Describe the bug**
 - Which OS do you encounter the bug on?
 
 Fedora Rawhide 
 
 - A clear and concise description of what the bug is.
 
 Failing to build the kernel
 
 - Attach the contents of your logs folder by drag & dropping each file into this message.

```bash
❯ sudo ./install.sh install
[sudo] password for romvnly: 
 -> Using script
Defaulting to kernel.org git mirror
Which linux distribution are you running ?
if it's not on the list, chose the closest one to it: Fedora/Suse for RPM, Ubuntu/Debian for DEB
    0) Debian
    1) Fedora
    2) Suse
    3) Ubuntu
    4) Gentoo
    5) Generic
[0-5]: 1
Selected: Fedora
 -> Which kernel version do you want to install?
    0) v6.10-rc7
  > 1) v6.9.8
    2) v6.8.12
    3) v6.7.12
    4) v6.6.38
    5) v6.5.13
    6) v6.4.16
    7) v6.1.97
    8) v5.15.162
    9) Another (no longer maintained upstream)
[0-9]: 0
Selected: v6.10-rc7
 -> Which CPU sched variant do you want to build/install?
 -> Project C (pds) / BMQ (bmq) is usually a good balance for gaming.
 -> Select "eevdf" (linux kernel's default) if unsure.
  > 0) EEVDF (Earliest Eligible Virtual Deadline First scheduler) Linux kernel's default for ≥ 6.6
[0-0]: 0
Selected: EEVDF (Earliest Eligible Virtual Deadline First scheduler) Linux kernel's default for ≥ 6.6
Which compiler do you want to use?
  > 0) GCC (recommended)
    1) Clang/LLVM
[0-1]: 0
Selected: GCC (recommended)
github.com	https://github.com/gregkh/linux.git (fetch)
github.com	https://github.com/gregkh/linux.git (push)
torvalds	https://github.com/torvalds/linux.git (fetch)
torvalds	https://github.com/torvalds/linux.git (push)
googlesource.com	https://kernel.googlesource.com/pub/scm/linux/kernel/git/stable/linux-stable (fetch)
googlesource.com	https://kernel.googlesource.com/pub/scm/linux/kernel/git/stable/linux-stable (push)
kernel.org	https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git (fetch)
kernel.org	https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git (push)
torvalds	https://github.com/torvalds/linux.git (fetch)
torvalds	https://github.com/torvalds/linux.git (push)
 -> Fetching tag: v6.10-rc7 from mirror kernel.org
remote: Enumerating objects: 6052, done.
remote: Counting objects: 100% (3415/3415), done.
remote: Compressing objects: 100% (520/520), done.
remote: Total 1538 (delta 1462), reused 1069 (delta 1014), pack-reused 0
Receiving objects: 100% (1538/1538), 288.83 KiB | 2.33 MiB/s, done.
Resolving deltas: 100% (1462/1462), completed with 1456 local objects.
From https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux
 * [new tag]             v6.10-rc7  -> v6.10-rc7
 -> Checking out tag: v6.10-rc7
 ->    in the work folder: /home/romvnly/linux-tkg//linux-src-git
Preparing worktree (detached HEAD 256abd8e5)
Updating files: 100% (84963/84963), done.
HEAD is now at 256abd8e5 Linux 6.10-rc7
 -> Installing dependencies
Updating and loading repositories:
 Fedora - Rawhide - Developmental packages for the next Fedora release                                                                                                      100% |  17.5 KiB/s |  10.6 KiB |  00m01s
 RPM Fusion for Fedora rawhide - Free tainted                                                                                                                               100% |   7.6 KiB/s |   2.4 KiB |  00m00s
 Fedora - Rawhide - Developmental packages for the next Fedora release                                                                                                      100% |   2.2 MiB/s |   4.5 MiB |  00m02s
Repositories loaded.
Package "perl-4:5.40.0-509.fc41.x86_64" is already installed.
Package "bison-3.8.2-7.fc40.x86_64" is already installed.
Package "ccache-4.10-1.fc41.x86_64" is already installed.
Package "dwarves-1.27-1.fc41.x86_64" is already installed.
Package "elfutils-devel-0.191-7.fc41.x86_64" is already installed.
Package "elfutils-libelf-devel-0.191-7.fc41.x86_64" is already installed.
Package "fedora-packager-0.6.0.7-9.fc40.noarch" is already installed.
Package "fedpkg-1.45-1.fc41.noarch" is already installed.
Package "flex-2.6.4-16.fc40.x86_64" is already installed.
Package "gcc-c++-14.1.1-7.fc41.x86_64" is already installed.
Package "git-2.45.2-1.fc41.x86_64" is already installed.
Package "libXi-devel-1.8.1-5.fc40.x86_64" is already installed.
Package "lz4-1.9.4-6.fc40.x86_64" is already installed.
Package "make-1:4.4.1-6.fc40.x86_64" is already installed.
Package "ncurses-devel-6.4-12.20240127.fc40.x86_64" is already installed.
Package "openssl-1:3.2.2-3.fc41.x86_64" is already installed.
Package "openssl-devel-1:3.2.2-3.fc41.x86_64" is already installed.
Package "perl-devel-4:5.40.0-509.fc41.x86_64" is already installed.
Package "perl-generators-1.16-5.fc40.noarch" is already installed.
Package "pesign-116-3.fc40.x86_64" is already installed.
Package "python3-devel-3.13.0~b3-2.fc41.x86_64" is already installed.
Package "qt5-qtbase-devel-5.15.14-1.fc41.x86_64" is already installed.
Package "rpm-build-4.19.92-2.fc41.x86_64" is already installed.
Package "rpmdevtools-9.6-7.fc40.noarch" is already installed.
Package "schedtool-1.3.0-30.fc40.x86_64" is already installed.
Package "zstd-1.5.6-1.fc41.x86_64" is already installed.
Package "bc-1.07.1-21.fc40.x86_64" is already installed.
Package "rsync-3.3.0-2.fc41.x86_64" is already installed.

Package                                                                 Arch             Version                                                                  Repository                                    Size
Upgrading:
 fedpkg                                                                 noarch           1.45-2.fc41                                                              rawhide                                  322.9 KiB
   replacing fedpkg                                                     noarch           1.45-1.fc41                                                              rawhide                                  322.2 KiB

Transaction Summary:
 Upgrading:         1 packages
 Replacing:         1 packages

Total size of inbound packages is 119 KiB. Need to download 119 KiB.
After this operation 750 B will be used (install 323 KiB, remove 322 KiB).
[1/1] fedpkg-0:1.45-2.fc41.noarch                                                                                                                                           100% | 255.7 KiB/s | 118.6 KiB |  00m00s
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
[1/1] Total                                                                                                                                                                 100% | 116.1 KiB/s | 118.6 KiB |  00m01s
Running transaction
[1/4] Verify package files                                                                                                                                                  100% |   1.0 KiB/s |   1.0   B |  00m00s
[2/4] Prepare transaction                                                                                                                                                   100% |   4.0   B/s |   2.0   B |  00m00s
[3/4] Upgrading fedpkg-0:1.45-2.fc41.noarch                                                                                                                                 100% |  13.5 MiB/s | 331.4 KiB |  00m00s
[4/4] Erasing fedpkg-0:1.45-1.fc41.noarch                                                                                                              100% [==================] | 142.0   B/s |  50.0   B | -00m00s
>>> Running trigger-install scriptlet: glibc-common-0:2.39.9000-30.fc41.x86_64warning: posix.fork(): .fork(), .exec(), .wait() and .redirect2null() are deprecated, use rpm.execute() instead
warning: posix.wait(): .fork(), .exec(), .wait() and .redirect2null() are deprecated, use rpm.execute() instead
[4/4] Erasing fedpkg-0:1.45-1.fc41.noarch                                                                                                                                   100% |  10.0   B/s |  50.0   B |  00m05s
>>> Running trigger-install scriptlet: glibc-common-0:2.39.9000-30.fc41.x86_64
>>> Stop trigger-install scriptlet: glibc-common-0:2.39.9000-30.fc41.x86_64
>>> Running trigger-install scriptlet: man-db-0:2.12.1-1.fc41.x86_64
>>> Stop trigger-install scriptlet: man-db-0:2.12.1-1.fc41.x86_64
>>> Running trigger-post-uninstall scriptlet: man-db-0:2.12.1-1.fc41.x86_64
>>> Stop trigger-post-uninstall scriptlet: man-db-0:2.12.1-1.fc41.x86_64
Complete!
 -> Using Arch patches
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 29004  100 29004    0     0  97140      0 --:--:-- --:--:-- --:--:-- 97328
 -> Applying graysky's cpu opts patch
 -> Applying clear linux patches
 -> Applying glitched base patch
 -> Skipping patch 0003-glitched-base-nonrt.patch...
         (unavailable for this kernel version)
 -> RPM: fixing spec generator
 -> Applying misc additions patch
 -> Skipping patch 0001-mm-Support-soft-dirty-flag-reset-for-VA-range.patch...
         (unavailable for this kernel version)
 -> Skipping patch 0002-mm-Support-soft-dirty-flag-read-with-reset.patch...
         (unavailable for this kernel version)
 -> Applying Glitched EEVDF additions patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  664k  100  664k    0     0   903k      0 --:--:-- --:--:-- --:--:--  902k
 -> Applying Sched_ext support patch
 -> Applying Glitched CFS/EEVDF patch
 -> Using archlinux's default config file for kernel 6.10
 -> Please select the desired CPU micro-architecture
    0) native_amd
    1) native_intel
  > 2) generic_cpu
    3) generic_cpu2
    4) generic_cpu3
    5) generic_cpu4
    6) k8
    7) k8sse3
    8) k10
    9) barcelona
    10) bobcat
    11) jaguar
    12) bulldozer
    13) piledriver
    14) steamroller
    15) excavator
    16) zen
    17) zen2
    18) zen3
    19) zen4
    20) mpsc
    21) atom
    22) core2
    23) nehalem
    24) westmere
    25) bonnell
    26) silvermont
    27) sandybridge
    28) ivybridge
    29) haswell
    30) broadwell
    31) skylake
    32) skylakex
    33) cannonlake
    34) icelake
    35) goldmont
    36) goldmontplus
    37) cascadelake
    38) emeraldrapids
    39) cooperlake
    40) tigerlake
    41) sapphirerapids
    42) rocketlake
    43) alderlake
    44) raptorlake
    45) meteorlake
[0-45]: 40
Selected: tigerlake
 -> Which kernel interrupt timer frequency would you like to use ?
 -> Higher Hz means lower latency (but higher overhead / less throughput). So it's a double edged sword
 -> Pick default (aka press enter with empty input) if unsure
    0) 100 Hz
    1) 250 Hz
    2) 300 Hz
    3) 500 Hz
    4) 750 Hz
  > 5) 1000 Hz
[0-5]: 5
Selected: 1000 Hz
Use CattaRappa mode (Tickless/Dynticks) ?
    0) No, use periodic ticks.
    1) Yes, full tickless baby!
       Full tickless can give higher performances in case you use isolation of CPUs for task, in other cases it behaves as Idle.
  > 2) Just tickless idle plz.
        Just tickless idle can perform better with some platforms (mostly AMD) or CPU schedulers (mostly MuQSS).
[0-2]: 2
Selected: Just tickless idle plz.
        Just tickless idle can perform better with some platforms (mostly AMD) or CPU schedulers (mostly MuQSS).

Use ACS override patch?
https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF#Bypassing_the_IOMMU_groups_.28ACS_override_patch.29
    > N/y : y
 -> Patching ACS override
 -> Import OpenRGB patch

Enable android modules for use with Waydroid?
https://waydro.id
    > N/y : y
Press enter to continue...
 -> Setting config
  HOSTCC  scripts/basic/fixdep
  HOSTCC  scripts/kconfig/conf.o
  HOSTCC  scripts/kconfig/confdata.o
  HOSTCC  scripts/kconfig/expr.o
  LEX     scripts/kconfig/lexer.lex.c
  YACC    scripts/kconfig/parser.tab.[ch]
  HOSTCC  scripts/kconfig/lexer.lex.o
  HOSTCC  scripts/kconfig/menu.o
  HOSTCC  scripts/kconfig/parser.tab.o
  HOSTCC  scripts/kconfig/preprocess.o
  HOSTCC  scripts/kconfig/symbol.o
  HOSTCC  scripts/kconfig/util.o
  HOSTLD  scripts/kconfig/conf
.config:11494:warning: override: DEBUG_INFO_DWARF5 changes choice state
#
# configuration written to .config
#
 -> SELinux activation for Fedora

*Optional* For advanced users - Do you want to use make menuconfig or nconfig
to configure the kernel before building it?
If you do, make sure your terminal is currently
at least 19 lines by 80 columns large or you'll get an error :D
    > 0. nope
      1. menuconfig
      2. nconfig
      3. xconfig
      choice[0-3?]: 0
 -> Configuration done.
 -> Enabled ccache
 -> Add patched files to the diff.patch
 -> Building kernel RPM packages
  SYNC    include/config/auto.conf
  UPD     include/config/kernel.release
  GEN     rpmbuild/SPECS/kernel.spec
rpmbuild -bb rpmbuild/SPECS/kernel.spec --define='_topdir /home/romvnly/linux-tkg/linux-src-git/rpmbuild' --target x86_64-linux --build-in-place --noprep --define='_smp_mflags %{nil}' $(rpm -q rpm >/dev/null 2>&1 || echo --nodeps) --define '_topdir /home/romvnly/linux-tkg//linux-src-git/rpmbuild'
Building target platforms: x86_64-linux
Building for target x86_64-linux
warning: %source_date_epoch_from_changelog is set, but %changelog has no entries to take a date from
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.OgV16X
+ umask 022
+ cd /home/romvnly/linux-tkg//linux-src-git/rpmbuild/BUILD/kernel-6.10.0_rc7_tkg_eevdf-build
+ cd kernel-6.10.0_rc7_tkg_eevdf
/var/tmp/rpm-tmp.OgV16X: line 36: cd: kernel-6.10.0_rc7_tkg_eevdf: No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.OgV16X (%build)

RPM build warnings:
    %source_date_epoch_from_changelog is set, but %changelog has no entries to take a date from

RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.OgV16X (%build)
make[2]: *** [scripts/Makefile.package:71: binrpm-pkg] Error 1
make[1]: *** [/home/romvnly/linux-tkg/linux-src-git/Makefile:1558: binrpm-pkg] Error 2
make: *** [Makefile:240: __sub-make] Error 2
 -> exit cleanup done


linux-tkg on  master [?] took 1m6s 
```

**Comment by BrycensRanch** [comment]
Hi, sorry for the slow response.

Yes.

```bash
❯ sudo ./install.sh install
 -> Using script
Defaulting to kernel.org git mirror
Which linux distribution are you running ?
if it's not on the list, chose the closest one to it: Fedora/Suse for RPM, Ubuntu/Debian for DEB
    0) Debian
    1) Fedora
    2) Suse
    3) Ubuntu
    4) Gentoo
    5) Generic
[0-5]: 1
Selected: Fedora
 -> Which kernel version do you want to install?
  > 0) v6.10
    1) v6.9.9
    2) v6.8.12
    3) v6.7.12
    4) v6.6.40
    5) v6.5.13
    6) v6.4.16
    7) v6.1.99
    8) v5.15.162
    9) Another (no longer maintained upstream)
[0-9]: 1
Selected: v6.9.9
 -> Which CPU sched variant do you want to build/install?
 -> Project C (pds) / BMQ (bmq) is usually a good balance for gaming.
 -> Select "eevdf" (linux kernel's default) if unsure.
  > 0) EEVDF (Earliest Eligible Virtual Deadline First scheduler) Linux kernel's default for ≥ 6.6
    1) BORE (Burst-Oriented Response Enhancer) CPU Scheduler
    2) Project C / PDS
    3) Project C / BMQ
[0-3]: 0
Selected: EEVDF (Earliest Eligible Virtual Deadline First scheduler) Linux kernel's default for ≥ 6.6
Which compiler do you want to use?
  > 0) GCC (recommended)
    1) Clang/LLVM
[0-1]: 0
Selected: GCC (recommended)
github.com	https://github.com/gregkh/linux.git (fetch)
github.com	https://github.com/gregkh/linux.git (push)
torvalds	https://github.com/torvalds/linux.git (fetch)
torvalds	https://github.com/torvalds/linux.git (push)
googlesource.com	https://kernel.googlesource.com/pub/scm/linux/kernel/git/stable/linux-stable (fetch)
googlesource.com	https://kernel.googlesource.com/pub/scm/linux/kernel/git/stable/linux-stable (push)
kernel.org	https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git (fetch)
kernel.org	https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git (push)
torvalds	https://github.com/torvalds/linux.git (fetch)
torvalds	https://github.com/torvalds/linux.git (push)
 -> Fetching tag: v6.9.9 from mirror kernel.org
remote: Enumerating objects: 36679, done.
remote: Counting objects: 100% (24997/24997), done.
remote: Compressing objects: 100% (12551/12551), done.
remote: Total 12797 (delta 11568), reused 759 (delta 201), pack-reused 0
Receiving objects: 100% (12797/12797), 4.79 MiB | 4.50 MiB/s, done.
Resolving deltas: 100% (11568/11568), completed with 11420 local objects.
From https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux
 * [new tag]             v6.9.9     -> v6.9.9
 -> Checking out tag: v6.9.9
 ->    in the work folder: /home/romvnly/linux-tkg//linux-src-git
Preparing worktree (detached HEAD 28fdf4518)
Updating files: 100% (84343/84343), done.
HEAD is now at 28fdf4518 Linux 6.9.9
 -> Installing dependencies
Updating and loading repositories:
Repositories loaded.
Package "perl-4:5.40.0-509.fc41.x86_64" is already installed.
Package "bison-3.8.2-7.fc40.x86_64" is already installed.
Package "ccache-4.10.1-1.fc41.x86_64" is already installed.
Package "dwarves-1.27-1.fc41.x86_64" is already installed.
Package "elfutils-devel-0.191-7.fc41.x86_64" is already installed.
Package "elfutils-libelf-devel-0.191-7.fc41.x86_64" is already installed.
Package "fedora-packager-0.6.0.7-9.fc40.noarch" is already installed.
Package "fedpkg-1.45-2.fc41.noarch" is already installed.
Package "flex-2.6.4-16.fc40.x86_64" is already installed.
Package "gcc-c++-14.1.1-7.fc41.x86_64" is already installed.
Package "git-2.45.2-1.fc41.x86_64" is already installed.
Package "libXi-devel-1.8.1-5.fc40.x86_64" is already installed.
Package "lz4-1.9.4-6.fc40.x86_64" is already installed.
Package "make-1:4.4.1-6.fc40.x86_64" is already installed.
Package "ncurses-devel-6.4-12.20240127.fc40.x86_64" is already installed.
Package "openssl-1:3.2.2-4.fc41.x86_64" is already installed.
Package "openssl-devel-1:3.2.2-4.fc41.x86_64" is already installed.
Package "perl-devel-4:5.40.0-509.fc41.x86_64" is already installed.
Package "perl-generators-1.16-5.fc40.noarch" is already installed.
Package "pesign-116-3.fc40.x86_64" is already installed.
Package "python3-devel-3.13.0~b3-2.fc41.x86_64" is already installed.
Package "qt5-qtbase-devel-5.15.14-5.fc41.x86_64" is already installed.
Package "rpm-build-4.19.92-2.fc41.x86_64" is already installed.
Package "rpmdevtools-9.6-7.fc40.noarch" is already installed.
Package "schedtool-1.3.0-30.fc40.x86_64" is already installed.
Package "zstd-1.5.6-1.fc41.x86_64" is already installed.
Package "bc-1.07.1-21.fc40.x86_64" is already installed.
Package "rsync-3.3.0-2.fc41.x86_64" is already installed.

Nothing to do.
 -> Using Arch patches
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 29004  100 29004    0     0   174k      0 --:--:-- --:--:-- --:--:--  173k
 -> Applying graysky's cpu opts patch
 -> Applying clear linux patches
 -> Applying glitched base patch
 -> Skipping patch 0003-glitched-base-nonrt.patch...
         (unavailable for this kernel version)
 -> RPM: fixing spec generator
 -> Applying misc additions patch
 -> Skipping patch 0001-mm-Support-soft-dirty-flag-reset-for-VA-range.patch...
         (unavailable for this kernel version)
 -> Skipping patch 0002-mm-Support-soft-dirty-flag-read-with-reset.patch...
         (unavailable for this kernel version)
 -> Applying Glitched EEVDF additions patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  704k  100  704k    0     0  1079k      0 --:--:-- --:--:-- --:--:-- 1080k
 -> Applying Sched_ext support patch
 -> Applying Glitched CFS/EEVDF patch
 -> Using archlinux's default config file for kernel 6.9
 -> Please select the desired CPU micro-architecture
    0) native_amd
    1) native_intel
  > 2) generic_cpu
    3) generic_cpu2
    4) generic_cpu3
    5) generic_cpu4
    6) k8
    7) k8sse3
    8) k10
    9) barcelona
    10) bobcat
    11) jaguar
    12) bulldozer
    13) piledriver
    14) steamroller
    15) excavator
    16) zen
    17) zen2
    18) zen3
    19) zen4
    20) mpsc
    21) atom
    22) core2
    23) nehalem
    24) westmere
    25) bonnell
    26) silvermont
    27) sandybridge
    28) ivybridge
    29) haswell
    30) broadwell
    31) skylake
    32) skylakex
    33) cannonlake
    34) icelake
    35) goldmont
    36) goldmontplus
    37) cascadelake
    38) emeraldrapids
    39) cooperlake
    40) tigerlake
    41) sapphirerapids
    42) rocketlake
    43) alderlake
    44) raptorlake
    45) meteorlake
[0-45]: 40
Selected: tigerlake
 -> Which kernel interrupt timer frequency would you like to use ?
 -> Higher Hz means lower latency (but higher overhead / less throughput). So it's a double edged sword
 -> Pick default (aka press enter with empty input) if unsure
    0) 100 Hz
    1) 250 Hz
    2) 300 Hz
    3) 500 Hz
    4) 750 Hz
  > 5) 1000 Hz
[0-5]: 5
Selected: 1000 Hz
Use CattaRappa mode (Tickless/Dynticks) ?
    0) No, use periodic ticks.
    1) Yes, full tickless baby!
       Full tickless can give higher performances in case you use isolation of CPUs for task, in other cases it behaves as Idle.
  > 2) Just tickless idle plz.
        Just tickless idle can perform better with some platforms (mostly AMD) or CPU schedulers (mostly MuQSS).
[0-2]: 2
Selected: Just tickless idle plz.
        Just tickless idle can perform better with some platforms (mostly AMD) or CPU schedulers (mostly MuQSS).

Use ACS override patch?
https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF#Bypassing_the_IOMMU_groups_.28ACS_override_patch.29
    > N/y : y
 -> Patching ACS override
 -> Import OpenRGB patch

Enable android modules for use with Waydroid?
https://waydro.id
    > N/y : y
Press enter to continue...
 -> Setting config
  HOSTCC  scripts/basic/fixdep
  HOSTCC  scripts/kconfig/conf.o
  HOSTCC  scripts/kconfig/confdata.o
  HOSTCC  scripts/kconfig/expr.o
  LEX     scripts/kconfig/lexer.lex.c
  YACC    scripts/kconfig/parser.tab.[ch]
  HOSTCC  scripts/kconfig/lexer.lex.o
  HOSTCC  scripts/kconfig/menu.o
  HOSTCC  scripts/kconfig/parser.tab.o
  HOSTCC  scripts/kconfig/preprocess.o
  HOSTCC  scripts/kconfig/symbol.o
  HOSTCC  scripts/kconfig/util.o
  HOSTLD  scripts/kconfig/conf
.config:11445:warning: override: DEBUG_INFO_DWARF5 changes choice state
#
# configuration written to .config
#
 -> SELinux activation for Fedora

*Optional* For advanced users - Do you want to use make menuconfig or nconfig
to configure the kernel before building it?
If you do, make sure your terminal is currently
at least 19 lines by 80 columns large or you'll get an error :D
    > 0. nope
      1. menuconfig
      2. nconfig
      3. xconfig
      choice[0-3?]: 0
 -> Configuration done.
 -> Enabled ccache
 -> Add patched files to the diff.patch
 -> Building kernel RPM packages
  SYNC    include/config/auto.conf
  UPD     include/config/kernel.release
  GEN     rpmbuild/SPECS/kernel.spec
rpmbuild -bb rpmbuild/SPECS/kernel.spec --define='_topdir /home/romvnly/linux-tkg/linux-src-git/rpmbuild' --target x86_64-linux --build-in-place --noprep --define='_smp_mflags %{nil}' $(rpm -q rpm >/dev/null 2>&1 || echo --nodeps) --define '_topdir /home/romvnly/linux-tkg//linux-src-git/rpmbuild'
Building target platforms: x86_64-linux
Building for target x86_64-linux
warning: %source_date_epoch_from_changelog is set, but %changelog has no entries to take a date from
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.tpcQoU
+ umask 022
+ cd /home/romvnly/linux-tkg//linux-src-git/rpmbuild/BUILD/kernel-6.9.9_tkg_eevdf-build
+ cd kernel-6.9.9_tkg_eevdf
/var/tmp/rpm-tmp.tpcQoU: line 36: cd: kernel-6.9.9_tkg_eevdf: No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.tpcQoU (%build)

RPM build warnings:
    %source_date_epoch_from_changelog is set, but %changelog has no entries to take a date from

RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.tpcQoU (%build)
make[2]: *** [scripts/Makefile.package:71: binrpm-pkg] Error 1
make[1]: *** [/home/romvnly/linux-tkg/linux-src-git/Makefile:1549: binrpm-pkg] Error 2
make: *** [Makefile:240: __sub-make] Error 2
 -> exit cleanup done


linux-tkg on  master [?] took 50s 
```


[shell-output.log.txt](https://github.com/user-attachments/files/16255146/shell-output.log.txt)
[customization.cfg.txt](https://github.com/user-attachments/files/16255152/customization.cfg.txt)


[prepare.log.txt](https://github.com/user-attachments/files/16255148/prepare.log.txt)

**Comment by AdelKS** [comment]
Hey can you try again with #1014

**Comment by BrycensRanch** [comment]
I accidentally destroyed my Fedora Rawhide installation this morning, sorry. I only have my root directory from 5 months ago and I doubt it'll upgrade when I get my Linux installation booting again.

**Comment by BrycensRanch** [comment]
You thought I was DEAD? You thought I wouldn't make it out of grub cli? 
![image](https://github.com/user-attachments/assets/a149d7ae-4a53-499c-98dc-39a3499125cf)


Well, I've got NEWS. It seems like this bug was fixed with #1014 . The kerne, is currently compiling and I will update once it's done. Cheers!

**Comment by BrycensRanch** [comment]
The build completed successfully. 🥲

I guess this is where this issue ends. 😭

It was a long and hard journey 😔
